src/net.cpp

```diff
@@ -2189,7 +2189,30 @@ void CConnman::ThreadDNSAddressSeed()
     std::vector<std::string> seeds = m_params.DNSSeeds();
     Shuffle(seeds.begin(), seeds.end(), rng);
     int seeds_right_now = 0; // Number of seeds left before testing if we have enough connections
-    int found = 0;
+    int target_outbound_connections = 2;
```

Comment:
Is there a particular rationale for this chosen target of 2?
nit: Recommend adding comment with rationale (perhaps I overlooked this).

Reply:
We are just keeping it consistent with what `nRelevant` used to be